pkgs/development/python-modules/scipy/default.nix

```diff
@@ -100,6 +103,13 @@ buildPythonPackage {
     pkg-config
     wheel
     setuptools
+  ] ++ lib.optionals stdenv.isDarwin [
+    # A shim xcrun executable, to fool upstream's meson.build that we have or
+    # on a macos with new enough SDK. We don't have the real executable
+    # available as a real package in Nixpkgs.
+    (writeShellScriptBin "xcrun" ''
+      echo "13.3"
+    '')
```

Comment:
Does `(xcbuild.override { sdkVer = "13.3"; })` work?

Reply:
> Does `(xcbuild.override { sdkVer = "13.3"; })` work?

I'm not sure it will evaluate, [this `xcbuild`](https://github.com/NixOS/nixpkgs/blob/staging-next/pkgs/development/tools/xcbuild/default.nix) expression doesn't accept an `sdkVer` argument, and according to `git grep` there are no places where such overrides are used.

Reply:
The argument is defined [in the wrapper](https://github.com/NixOS/nixpkgs/blob/staging-next/pkgs/development/tools/xcbuild/wrapper.nix), and you'll see there's an `xcrun = writeTextFile something` just below.

Reply:
> The argument is defined [in the wrapper](https://github.com/NixOS/nixpkgs/blob/staging-next/pkgs/development/tools/xcbuild/wrapper.nix?rgh-link-date=2024-07-01T07%3A17%3A55Z), and you'll see there's an `xcrun = writeTextFile something` just below.

OK great, indeed it is better then my script.